### DISCUSSION TITLE
Send humidity values to Bosch Heatpump (for cooling station control)

### DISCUSSION THREAD

**Discussion by lemked** [emsesp/EMS-ESP32#1648] [Q&A]
Hi there,

I have a brand-new Bosch Compress 7800i LW heatpump in our newly built house. During the pre-setup, the Bosch technician told me that I have to install a Bosch CR 10H unit somewhere in our living rooms for measuring humidity and send them to the heatpump (via EMS bus cable). Apparently this is needed to minimize the risk of having water condensating. I was wondering if that extra unit is really needed and whether I couldn't simply provide the humidity value using one of my many Thermostats that I have installed anyway (which do support humidity measurement).

So is there a way to send humidity values to the heat pump via the EMS-ESP32 firmware? I was anyway thinking about buying a BBQKees Gateway for monitoring of the heat pump. If I could use the Gateway to provide the humidity values to the heat pump, that'd be a nice feature on top!

**Comment by MichScha** [comment]
Just a few discuasions down: https://github.com/emsesp/EMS-ESP32/discussions/1287

**Comment by MichaelDvP** [comment]
The #1287 is about emulating a RC100H for remote temperature and humidiy. We know from #826 that CR10 (Bosch) is identical with RC100 (Buderus), sharing same product id. Maybe the CR10H is also identical to RC100H, than it works as is.  
If not, we have to wait to see a log with a real CR10H attached to see the difference.

**Comment by lemked** [comment]
Thanks for quick replies. I'll report back here as soon as the hardware is running.

**Reply by Scholdan** [reply]
Did you every get it working?

**Reply by lemked** [reply]
I have the gateway running since 2 weeks, but haven't tried yet emulating the thermostat because the cooling function of the heat pump isn't working properly yet. Will have a Bosch technician coming next week to check. After that, I may look into the emulation again.
